packages/manager/modules/pci/src/projects/project/storages/blocks/help/snapshot.html

```diff
@@ -1,12 +1,14 @@
-<p>
-    <span class="d-block" data-translate="pci_projects_project_storages_blocks_help_snapshot_title"></span>
-    <span data-translate="pci_projects_project_storages_blocks_help_snapshot_description"></span>
-</p>
+<div class="pci-projects-storages-blocks-help-snapshot">
```

Comment:
Could you please stick with `indent_size = 4`?

Thanks!

Reply:
sure, done